sites/public/pages/eligibility/income.tsx

```diff
@@ -4,9 +4,70 @@ Prompts the user for their annual income.
 */
 import FormsLayout from "../../layouts/forms"
 import React from "react"
+import { FormCard } from "@bloom-housing/ui-components/src/blocks/FormCard"
+import { t } from "@bloom-housing/ui-components/src/helpers/translator"
+import { ProgressNav } from "@bloom-housing/ui-components/src/navigation/ProgressNav"
+import { ELIGIBILITY_ROUTE, ELIGIBILITY_SECTIONS } from "../../lib/constants"
+import { Form } from "@bloom-housing/ui-components/src/forms/Form"
+import { Button } from "@bloom-housing/ui-components/src/actions/Button"
+import { AppearanceStyleType, Select } from "@bloom-housing/ui-components"
+import { useRouter } from "next/router"
+import { useForm } from "react-hook-form"
 
 const EligibilityIncome = () => {
-  return <FormsLayout />
+  const router = useRouter()
+
+  const incomeRanges = ["below10k", "10kTo20k", "30kTo40k", "40kTo50k", "over50k"]
+
+  /* Form Handler */
+  const { handleSubmit, register } = useForm()
+  const onSubmit = () => {
+    // Not yet implemented.
+  }
+
+  return (
+    <FormsLayout>
+      <FormCard header={t("eligibility.progress.header")}>
+        <ProgressNav
+          currentPageSection={5}
+          completedSections={4}
+          labels={ELIGIBILITY_SECTIONS.map((label) => t(`eligibility.progress.sections.${label}`))}
+        />
+      </FormCard>
+      <FormCard>
+        <div className="form-card__lead pb-0">
+          <h2 className="form-card__title is-borderless">{t("eligibility.income.prompt")}</h2>
+        </div>
+        <Form onSubmit={handleSubmit(onSubmit)}>
```

Comment:
What user action prompts this `onSubmit` callback? (IIUC, the clicking the Button below triggers the `onClick` action)

Reply:
I believe buttons trigger onSubmit by default unless we pass in type="button". Clicking the button should trigger this as well.
I haven't set up all the form validation yet, but I'll need this [hook to validate the form](https://react-hook-form.com/api/useform).

Reply:
Gotcha, thanks. I'm wondering whether we want this `onSubmit` method at all (as opposed to doing everything, including form validation, in `onClick`)? Or vice versa and get rid of `onClick`?

(I may not be understanding the difference between `onClick` and `onSubmit` - I'm fine to keep both since it sounds like you know what you're doing and are maybe following an established pattern!)

Reply:
Oh I think validation is passed to the actual component like the "Select" below. See "validation" on line 49, and onSubmit is needed to trigger that and display any errors (I haven't added the errors yet).

Reply:
Huh okay, thanks for explaining!